### ISSUE TITLE
Method for using Vampire Analysis in Jupyter notebook

### ISSUE THREAD

**Issue by hhelmbre** [kukionfr/VAMPIRE_open#3]
Hi!

I have been using the VAMPIRE GUI, but for higher throughput image processing, I was wondering if there was a way to integrate vampire analysis in jupyter notebook.

I tried doing it myself, but I am running into issues due to the GUI Tkinter interface.

Any help would be appreciated!

**Comment by hhelmbre** [comment]
Okay so I have it working in Jupyter notebook (although I still have to use the GUI which isn't ideal). But I am trying to run your example data, and it seems like the software has stopped running the getboundary steps so the c1 pickle isn't being created for building or applying the model.

**Comment by kukionfr** [comment]
Hello!

Currently, we only have either the executable file or the source codes in py format, and both options require the GUI. 
I will take a look at how you could easily specify inputs and output of VAMPIRE to execute its core algorithm in Jupyter. 
Will give an update here if changes are made.

**Comment by hhelmbre** [comment]
That's okay with me for now. I figured out a faster way to use it then what we had before. But my second comment still stands that the current version is skipping the step of making a pickle so it isn't working. Are you having the same issue?

**Comment by Crescentz** [comment]
> That's okay with me for now. I figured out a faster way to use it then what we had before. But my second comment still stands that the current version is skipping the step of making a pickle so it isn't working. Are you having the same issue?

hi, I also want to use it in Jupyter, could you pls show your code ?

**Comment by kukionfr** [comment]
@Crescentz Yes, I will find a time this week to write an example jupyter notebook.
